beacon-chain/sync/subscriber.go

```diff
@@ -130,6 +131,17 @@ func (s *Service) registerSubscribers(epoch primitives.Epoch, digest [4]byte) {
 			digest,
 		)
 	}
+
+	// New Gossip Topic in Deneb
+	if epoch >= params.BeaconConfig().DenebForkEpoch {
+		s.subscribeStaticWithSubnets(
+			p2p.BlobSubnetTopicFormat,
+			s.validateBlob,   /* validator */
+			s.blobSubscriber, /* message handler */
+			digest,
+			params.BeaconConfig().MaxBlobsPerBlock,
```

Comment:
I think we should have a `BlobsSubnetCount` value in the network config for symmetry with `AttestationSubnetCount`.

Reply:
dbdf3851f